1.10/deploying-services/gpu.md

```diff
@@ -0,0 +1,180 @@
+---
+post_title: Using GPUs
+feature_maturity: preview
+menu_order: 110
+---
+
+DC/OS supports allocating GPUs (Graphics Processing Units) to your long-running DC/OS services. Adding GPUs to your services can dramatically accelerate big data workloads. 
+
+With GPU-based scheduling, you can share cluster resources for traditional and machine learning workloads, as well as dynamically allocate GPU resources inside those clusters and free them when needed. You can reserve GPU resources for the workloads that need them, or pool these GPU-enabled resources with the rest of the infrastructure for higher overall utilization.
+
+After installing DC/OS with GPUs enabled, you can specify GPUs in your application definitions with the `gpus` parameter.
+
+# Installing DC/OS with GPUs Enabled
+GPUs must be enabled during DC/OS installation. Follow the instructions below to enable GPUs based on your specific DC/OS deployment method.
+
+## Custom DC/OS Installation with GPUs
+
+1.  Install the [NVIDIA Management Library (NVML)](https://developer.nvidia.com/nvidia-management-library-nvml) on each node of your cluster that has GPUs. The minimum required NVIDIA driver version is 340.29. For detailed installation instructions, see the [Mesos GPU support documentation](http://mesos.apache.org/documentation/latest/gpu-support/#external-dependencies).
+1.  Install DC/OS using the [custom advanced installation instructions](/docs/1.10/installing/custom/advanced/). Here are the GPU-specific configuration parameters:
```

Comment:
no, you only need a new line in some circumstances (e.g. after a bulleted list).